plugins/kubernetes/test/fixtures/kubernetes/release_docs.yml

```diff
@@ -3,4 +3,8 @@ test_release_pod_1:
   kubernetes_release: test_release
   deploy_group: pod1
   replica_target: 2
+  limits_cpu: 1
+  requests_cpu: 1
+  limits_memory: 1
+  requests_memory: 1
```

Comment:
unrelated to this PR?

Reply:
removed ... my local hacks until next PR is merged :D